src/Call/Oneclick/InitPaymentRequest.php

```diff
@@ -45,11 +43,11 @@ public function __construct(
 		string $merchantId,
 		string $origPayId,
 		string $orderId,
-		int $totalAmount = null,
-		Currency $currency = null,
+		Cart $cart = null,
```

Comment:
Nevim, no, u všech ostatních volání se držíme stejných parametrů requestů jako má ČSOB api, navíc jinde kde je nějaký amount, to použít nejde protože tam zase není currency (refund, položky Cart...).
Já bych zůstal u dvou parametrů.